components/ngc-float-item-button.component.ts

```diff
@@ -78,6 +84,11 @@ export class NgcFloatItemButtonComponent {
   @Input() icon;
   @Input() content;
   @Output() clicked: EventEmitter<any> = new EventEmitter();
+  @Input() isDisabled = false;
   @ViewChild('elementref') elementref;
   @ViewChild('contentref') contentref;
+
+  emitClickEvent($event: Event): void {
+    this.clicked.emit($event);
```

Comment:
for what reason you create the `emitClickEvent()` method in the component but 

```
(click)="clicked.emit($event)"> 
```

Is more simplified and also works, right?

Reply:
I prefer to have a method with a name that implies what the function is going to do - and have the actual implementation inside the component's class, not the HTML.